package.json

```diff
@@ -77,6 +77,7 @@
     "keytar": "^3.0.0",
     "l20n": "^3.5.1",
     "lru_cache": "^1.0.0",
+    "node-forge": "^0.6.4",
```

Comment:
I did some little hack on [electron.](brave/electron#25) for 77c534a. I think maybe we can make some changes on electron to extract the detail attributes from certificate. It may be a long term and better solution and we can reduce the effect to integrate with `node-forge`